.github/workflows/linting.yaml

```diff
@@ -32,8 +32,8 @@ jobs:
         run: |
           uv sync --extra tests
 
-      - name: Remove .so files
-        run: find . -name "*.so" -delete
+      # - name: Remove .so files
```

Comment:
@johnrachwan123 Not sure if it's related to this or not :shrug:

Reply:
It seems [that](https://github.com/PrunaAI/pruna/pull/165/commits/1245fcd6e89f745090ce373be1f0b6705bbe4bec) was it; it's in your hand now @johnrachwan123